tests/test_utils.cpp

```diff
@@ -60,44 +60,27 @@ void printIssues(const libcellml::LoggerPtr &l, bool headings, bool causes, bool
     }
 }
 
-void prettyPrintModelToStdout(libcellml::ModelPtr &model, bool includeMaths)
-{
-    std::string spacer = "    ";
-
-    std::cout << " MODEL: '" << model->name() << "'";
-    if (model->id() != "") {
-        std::cout << ", id: '" << model->id() << "'";
-    }
-    std::cout << std::endl;
-
-    std::cout << spacer << "UNITS: " << model->unitsCount() << " custom units" << std::endl;
-    for (size_t u = 0; u < model->unitsCount(); ++u) {
-        std::cout << spacer << spacer << "[" << u << "]: " << model->units(u)->name() << std::endl;
-    }
+static const std::string INDENT = "    ";
 
-    std::cout << spacer << "COMPONENTS: " << model->componentCount() << " components" << std::endl;
-    for (size_t c = 0; c < model->componentCount(); ++c) {
-        auto component = model->component(c);
-        prettyPrintComponentToStdout(component, c, spacer + spacer, includeMaths);
-    }
-}
-
-void prettyPrintComponentToStdout(const libcellml::ComponentPtr &component, size_t const c, std::string const spacer, bool includeMaths)
+void printComponent(const libcellml::ComponentPtr &component, size_t c, const std::string &indent, bool includeMaths)
 {
-    std::string local = "    ";
+    if (c == -1) {
+        std::cout << "COMPONENT: '" << component->name() << "'";
+    } else {
+        std::cout << indent << "[" << c + 1 << "]: " << component->name();
+    }
 
-    std::cout << spacer << "[" << c << "]: " << component->name();
     if (component->id() != "") {
-        std::cout << " id: " << component->id();
+        std::cout << ", id: " << component->id();
     }
-    std::cout << std::endl;
 
-    std::cout << spacer << local << "VARIABLES: " << component->variableCount() << " variables" << std::endl;
+    std::cout << std::endl;
+    std::cout << indent << INDENT << "VARIABLES: " << component->variableCount() << " variables" << std::endl;
```

Comment:
Mixing use of 'indent' and 'INDENT' here and in other places.

Reply:
No, no mixing. `INDENT` is a constant used to indent while `indent` is an indent that we need for calling `printComponent()` recursively.